### PR TITLE
clr-boot-manager.zsh: Added zsh completion

### PR DESCRIPTION
**Pull request by lebensterben**

### CODE REVIEW

File: clr-boot-manager.zsh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+if [[ $state == subcmd ]]; then
+   local -a subcmds; subcmds=(
+     "version:Print the version and quit"
+     "report-booted:Report the current kernel as successfully booted"
+     "update:Perform post-update configuration of the system"
+     "set-timeout:Set the timeout to be used by the bootloader"
+     "get-timeout:Get the timeout to be used by the bootloader"
+     "set-kernel:Configure kernel to be used at next boot"
+     "list-kernels:Display currently selectable kernels to boot"
```
Comment (bryteise) [inline clr-boot-manager.zsh]: May want to add help to this list as it is a useful feature. Would you want to consider nesting support for help as well (clr-boot-manager help <tab>)? Similar comment for the bash PR.

Comment (lebensterben) [reply clr-boot-manager.zsh]: @bryteise

Comment (lebensterben) [reply clr-boot-manager.zsh]: Okay, I see. You mean `clr-boot-manager help [subcommand]`.
This is not documented I think. I read the source code to figure this out.

Comment (bryteise) [reply clr-boot-manager.zsh]: Yea, I need to fix the man pages X(.

**Pull request closed without merging**